flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdUpsertKeysTest.scala

```diff
@@ -188,7 +188,7 @@ class FlinkRelMdUpsertKeysTest extends FlinkRelMdHandlerTestBase {
       rank => assertEquals(toBitSet(Array(0)), mq.getUpsertKeys(rank).toSet)
     }
 
-    Array(logicalRowNumber, flinkLogicalRowNumber, streamRowNumber)
+    Array(logicalWindow, logicalRowNumber, flinkLogicalRowNumber, streamRowNumber)
```

Comment:
I'm wondering why the existing tests seem already have some ability to infer `row_number` as a part of unique key?